api/v1beta1/clusterclass_types.go

```diff
@@ -336,12 +336,14 @@ type ClusterClassPatch struct {
 	// +optional
 	EnabledIf *string `json:"enabledIf,omitempty"`
 
-	// Definitions define the patches inline.
+	// Definitions define inline patches.
 	// Note: Patches will be applied in the order of the array.
+	// Note: Exactly one of Definitions or External has to be set.
```

Comment:
```suggestion
	// Note: Exactly one of Definitions or External must be set.
```

Maybe slightly clearer? Very weak nit 🙂

Reply:
Will do no problem :)